### DISCUSSION TITLE
Is this closed source?

### DISCUSSION THREAD

**Discussion by gorenbk** [helloSystem/hello#521] [Q&A] [answered]

**Comment by probonopd** [comment, marked as answer]
This is open source. You can learn more about it here: https://hellosystem.github.io/docs/

And you can find the source code here: https://github.com/helloSystem/

**Comment by gorenbk** [comment]
All I can find is the ISO file or the branding things.

**Comment by probonopd** [comment]
All code is there. Can I help you locate the code you are looking for? What exactly are you interested in?

**Reply by gorenbk** [reply]
Where is the code for the GUI?

**Reply by probonopd** [reply]
Which component do you mean by "the GUI"? The file manager? The menu? The utilities? They all have their own repository.

**Reply by gorenbk** [reply]
The GUI behind all that: for how any application is displayed.

**Reply by gorenbk** [reply]
Not for the background, menu, etc. Just for the border around the application that says it's name.

**Reply by probonopd** [reply]
This is the source code for the window decorations (like the title bar of each window): https://github.com/helloSystem/BreezeEnhanced
